datasets/jigsaw_toxicity_pred/jigsaw_toxicity_pred.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Comments from Jigsaw Toxic Comment Classification Kaggle Competition """
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import pandas as pd
+
+import datasets
+
+
+_DESCRIPTION = """\
+This dataset consists of a large number of Wikipedia comments which have been labeled by human raters for toxic behavior.
+"""
+
+_HOMEPAGE = "https://www.kaggle.com/c/jigsaw-toxic-comment-classification-challenge/data"
+
+_LICENSE = 'The "Toxic Comment Classification" dataset is released under CC0, with the underlying comment text being governed by Wikipedia\'s CC-SA-3.0.'
+
+
+class JigsawToxicityPred(datasets.GeneratorBasedBuilder):
+    """This is a dataset of comments from Wikipedia’s talk page edits which have been labeled by human raters for toxic behavior."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    @property
+    def manual_download_instructions(self):
+        return """\
+            To use jigsaw_toxicity_pred you have to download it manually from Kaggle: https://www.kaggle.com/c/jigsaw-toxic-comment-classification-challenge/data
+            You can manually download the data from it's homepage or use the Kaggle CLI tool (follow the instructions here: https://www.kaggle.com/docs/api)
+            Please extract all files in one folder and then load the dataset with:
+            `datasets.load_dataset('jigsaw_toxicity_pred', data_dir='/path/to/extracted/data/')`"""
+
+    def _info(self):
+
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # This defines the different columns of the dataset and their types
+            features=datasets.Features(
+                {
+                    "comment_text": datasets.Value("string"),
+                    "toxic": datasets.Value("float32"),
+                    "severe_toxic": datasets.Value("float32"),
+                    "obscene": datasets.Value("float32"),
+                    "threat": datasets.Value("float32"),
+                    "insult": datasets.Value("float32"),
+                    "identity_hate": datasets.Value("float32"),
```

Comment:
can you use the ClassLabel feature type for those fields ?
for example
```
"toxic": datasets.ClassLabel(names=["false", "true"])
```

to make it work you'll have to cast to `int` instead of `float` in _generate_examples